u/unixODBC/unixODBC-2.3.12-GCCcore-12.2.0.eb

```diff
@@ -0,0 +1,16 @@
+easyblock = 'ConfigureMake'
+
+name = 'unixODBC'
+version = '2.3.12'
+
+homepage = "https://www.unixodbc.org"
+description = """unixODBC provides a uniform interface between
+application and database driver"""
+
+toolchain = {'name': 'GCCcore', 'version': '12.2.0'}
```

Comment:
Hmm, this was built with `foss` in EasyBuild, but I have no idea why, so that's probably a bug to fix there...